rdrf/rdrf/templates/admin/notifications_disabled.html

```diff
@@ -0,0 +1,8 @@
+{% load i18n %}
+
+<p>
+  {% trans "Registry " %}'{{ registry.name }}'({{ registry.code }}){% trans " has disabled notifications that need to be enabled for registration to work" %}</p>
+<p>
+  {% trans "Current registration notifications belonging to the registry: " %}
```

Comment:
Can we change these to "<a>Click here</a> to edit the notifications belonging to this registry"?